docs/user-guide/node-selection/index.md

```diff
@@ -198,3 +198,215 @@ must be satisfied for the pod to schedule onto a node.
 
 For more information on inter-pod affinity/anti-affinity, see the design doc
 [here](https://github.com/kubernetes/kubernetes/blob/{{page.githubbranch}}/docs/design/podaffinity.md).
+
+## Taints and tolerations (beta feature)
+
+### Specifying taints and tolerations
+
+Node affinity, described earlier, allows a pod to "be attracted to" a set of nodes
+(either as a preference or a hard requirement). Taints are the opposite -- they allow
+a *node* to *repel* a set of pods.
+
+More concretely: *taints* go on nodes, and *tolerations* go on pods. A taint on a node
```

Comment:
I used a modified version of your suggested text as the second paragraph, which I think resolves this issue.